src/platform/baytrail/platform.c

```diff
@@ -65,11 +65,14 @@ static const struct sof_ipc_fw_ready ready
 	},
 	/* dspbox is for DSP initiated IPC, hostbox is for host initiated IPC */
 	.version = {
+	/* Add build number and timestamp for debug build */
+#ifdef DEBUG_BUILD
 		.build = SOF_BUILD,
 		.minor = SOF_MINOR,
```

Comment:
Are you sure about remove SOF version?
which means 1.0 or 1.1 is not needed?
I think you only need to remove date, time and build